e2e/basicProps.e2e.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-env detox/detox, jest/globals */
+import {
+  afterAllTestPlan,
+  beforeAllTestPlan,
+  waitForClearLogs,
+  waitForTestPlan,
+  waitForLogLabel,
+  waitForTap,
+} from './testPlan';
+
+describe('Basic Props', () => {
+  beforeAll(beforeAllTestPlan);
+  afterAll(afterAllTestPlan);
+
+  it('paused controls playback state', async () => {
+    await waitForTestPlan(`
```

Comment:
In a future pass, maybe something like a fluent DSL would be useful here. We could likely achieve similar readability, without the need to parse. Would also make it simpler to extend for more nuanced cases.